buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -782,6 +787,10 @@ private void startElasticsearchProcess() {
         reaper.registerPid(toString(), esProcess.pid());
     }
 
+    private Path getEffectiveDistroDir() {
+        return canUseSharedDistribution() ? getExtractedDistributionDir().toFile().listFiles()[0].toPath() : getDistroDir();
```

Comment:
Is there a reason we can't just change the implementation of `getDistroDir()` to return this value? In what scenario would we use both?

Reply:
fixed